tests/test_electric_source_sink.py

```diff
@@ -12,11 +12,21 @@
 
 
 class TestMILPElectricSourceSink(TestCase):
-    """Unit tests for the MILP test case of a source, a cable, a sink"""
-
     def test_source_sink(self):
-        """Test to verify if when a target it set that is more or less than
-        then max or min of the respective component it is capped. 1000"""
+        """
+        Tests for the MILP test case of a source, a cable, a sink.
+
+        Checks:
+        - Check that the caps set in the esdl work as intended
+        - Check that the consumed power is always>= 0.
+        - Check for energy conservation with consumed power, lost power and produced power.
+        - Check that the voltage drops over the line.
+
+        Missing:
+        The hardcoded stuff should be replaced.
```

Comment:
What is the problem with the hard coded values? I would like to know more about this.

Reply:
I just want to avoid as much as possible the maintenance effort of the code. With the hardcoded stuff we might have that functionality breaks when the logic is still fine, but more troublesome also the other way around